docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_indexscrollbar.htm

```diff
@@ -100,7 +102,11 @@ <h2><a id='manual-constructor0.6990520032122731'></a>How to create index scroll
          &lt;/ul&gt;
       &lt;/div&gt;
    &lt;/section&gt;
-   &lt;script&gt;
+</pre>
+
+<p>JS code:</p>
+
+<pre class="prettyprint">
```

Comment:
Related with this fix, don't we need to remove `&lt;/script&gt;` on line 165, too?

Also it looks like this "JS code" section is showing part of the whole code.
If so, what about `&lt;/div&gt;` on line 166?
It is closing of `<div id="pageIndexScrollbar" class="ui-page">` in line 78, I think.

Reply:
Yes, you are right... I will do fix. Thank you for review and attention.